session.go

```diff
@@ -100,7 +102,19 @@ func (cfg *SessionConfig) Validate() error {
 type Session struct {
 	cfg     SessionConfig
 	cluster *transport.Cluster
-	policy  transport.HostSelectionPolicy
+	hsp     atomic.Value // HostSelectionPolicy
```

Comment:
Should we keep the naming as it was? This is hsp -> policy.

Reply:
I changed policy to `hsp`, because you cannot have method and filed of the same name.
We won't really be using `hsp` anyways, we will always call `policy()` method to load it from `atomic.Value`.